.github/workflows/ci-cd.yml

```diff
@@ -3,7 +3,7 @@ name: vmware-cloud-director-ui-components
 on:
     push:
     pull_request:
-        branches: [master, a11y]
+        branches: [master, a11y, v2]
```

Comment:
I don't remember how are we going to version the branchec v2, v3 or to follow the versioning of vcd releases